CommunifyLibrary/Repository/MockRepository/MockUserRepository.cs

```diff
@@ -98,5 +99,15 @@ public Task<long> GetIdByEmailAsync(string email)
         {
             throw new NotImplementedException();
         }
+
+        public IQueryable<User> GetByUsername(string username)
+        {
+            throw new NotImplementedException();
```

Comment:
mockun amaci ne herseyde exception aticaksa? ya mocku sil yad abasit birsey cevir 2 dakkalik is